texastribune/donations
# Giving Tuesday 2024

#### What's this PR do?

Adds some updates for Giving Tuesday that falls on 12/2

#### Why are we doing this? How does it help us?

Membership drive

#### How should this be manually tested?

Review on https://donations-testing.herokuapp.com/donate
Just a general check on design, broken HTML etc. Emily is reviewing that this matches her spec separately

Review on https://waco-donations-testing-1119390b0075.herokuapp.com/donate
Confirm that nothing is changed here. I didn't notice that it disrupted anything on the waco site but let me know if I overlooked


#### How should this change be communicated to end users?

Deploy on 12/2 and let Emily know

#### Are there any smells or added technical debt to note?

Sounds like some of this might just be default independent of the drive. I tried to make it easy to undo with USE_THERMOMETER=False, but we'll see how much of that applies post-drive

#### What are the relevant tickets?

https://airtable.com/appyo1zuQd8f4hBVx/pagaqtKlLeA1VeYTv?detail=eyJwYWdlSWQiOiJwYWdsUzN3Y2tEdHB3elF6UCIsInJvd0lkIjoicmVjaHV2UWpHS1Y2cVBKcHciLCJzaG93Q29tbWVudHMiOmZhbHNlLCJxdWVyeU9yaWdpbkhpbnQiOm51bGx9

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:

* [x] *Get social share graphic*
* [x] *Have Emily I review*
